### PR TITLE
added 3 MoSo apps

### PR DESCRIPTION
**Pull request by vansterzzz**
IAM-1220, added 3 Mozilla Social apps

### CODE REVIEW

File: apps.yml [hunk 1]
```diff
@@ -4202,3 +4202,34 @@ apps:
...
+    display: false
+    logo: auth0.png
+    name: MoSo Invite Dashboard - Prod
+    op: auth0
+    url: https://prod.mozsoc-accts.prod.webservices.mozgcp.net/auth/callback
+- application:
+    authorized_groups:
+    - mozilliansorg_moso_invite_dashboard_stage-access
+    authorized_users: []
+    client_id: 4lgABkbb89J4OycvfsqkmzQTZCdsjyOt
```
Comment (ctbfourone) [inline apps.yml:4219]: This looks like the client_id for local dev, not stage.

File: apps.yml [hunk 2]
```diff
@@ -4202,3 +4202,34 @@ apps:
...
+    logo: auth0.png
+    name: MoSo Invite Dashboard - Stage
+    op: auth0
+    url: https://stage.mozsoc-accts.nonprod.webservices.mozgcp.net/auth/callback
+- application:
+    AAL: LOW
+    authorized_groups:
+    - everyone
+    authorized_users: []
+    client_id: 10SMTZkVHFSJRdddT43Uvyhi87dHY3MQ
```
Comment (ctbfourone) [inline apps.yml:4230]: And this looks like the client_id for stage, not local dev. :)

**Review by ctbfourone — CHANGES_REQUESTED** [review]
Looks good once you fixup these client ids.

**Review by ctbfourone — APPROVED** [review]

File: apps.yml (hunk 2 above)
Comment (vansterzzz) [reply apps.yml]: fixed, ty!

**Pull request merged**